app/finders/db_exports/known_locations.rb

```diff
@@ -17,7 +21,7 @@ class KnownLocations < CMSScanner::Finders::Finder
         def aggressive(opts = {})
           found = []
 
-          enumerate(potential_urls(opts), opts.merge(check_full_response: 200)) do |res|
+          enumerate(potential_urls(opts), opts.merge(check_full_response: [200, 206])) do |res|
```

Comment:
Could we use `valid_response_codes` here, rather than repeating the array?

Reply:
@alexsanford 

Thank you for the review !
I thought I should follow the other finders but using `valid_response_codes` is much better, I will fix it :)

Reply:
Great, thanks!